kmath-tensors/src/commonMain/kotlin/space/kscience/kmath/tensors/api/TensorStructure.kt

```diff
@@ -0,0 +1,5 @@
+package space.kscience.kmath.tensors.api
+
+import space.kscience.kmath.nd.MutableStructureND
+
+public typealias TensorStructure<T> = MutableStructureND<T>
```

Comment:
Why not `Tensor`?

Reply:
No objections here two both variants. Should be discussed.

Reply:
Yes, it should.